src/Codeception/Step/Comment.php

```diff
@@ -8,17 +8,17 @@ class Comment extends CodeceptionStep
 {
     public function __toString()
     {
-        return $this->getAction();
+        return (string)$this->getAction();
     }
 
     public function toString($maxLength)
     {
-        return $this->getAction();
+        return (string)$this->getAction();
```

Comment:
Why having a mandatory parameter $maxLength that is not use? I would be better to have toString() and getHumanizedAction() calling $this->__toString() that is doing the type casting.